WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AIToneVoice/AIToneVoiceView.swift

```diff
@@ -0,0 +1,103 @@
+import SwiftUI
+
+/// View to select AI tone and voice
+///
+struct AIToneVoiceView: View {
+    @StateObject private var viewModel: AIToneVoiceViewModel
+
+    @Environment(\.presentationMode) private var presentation
```

Comment:
👀 AFAIK `presentationMode` is [deprecated](https://developer.apple.com/documentation/swiftui/presentationmode) soon - we should switch to `dismiss`.

Reply:
Thanks for the tip. Updated in https://github.com/woocommerce/woocommerce-ios/pull/10754/commits/81d551bfc6fa7edbf82a8913de368ca0086708e2